addons/website/views/snippets/s_images_mosaic.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_images_mosaic" name="Images Mosaic">
+    <section class="s_images_mosaic pt72 pb72 o_cc o_cc5">
+        <div class="container">
+            <div class="row px-3">
+                <div class="col-lg-3">
```

Comment:
I find the text column a bit too crowded at the initial LG steps ( `> LG` it's ok tho 👍 ).
I know that you had to find "a compromise", but I'd trade a bit of visual result on very large screen for more clarity on regular/common devices. Wdyt? Could you test?  
 
| This PR: `col-lg-3` | Proposal: `col-lg-4` (+ remove sibling's `offset-lg-1`) |
|--------|--------|
| <img width="969" alt="image" src="https://github.com/user-attachments/assets/5ef296ad-5a5d-45d9-b4f0-40c88b9c255d"> | <img width="977" alt="image" src="https://github.com/user-attachments/assets/2e7fa182-69da-4e0d-a2d2-835febad07db"> |